src/main/java/tutorlink/parser/Parser.java

```diff
@@ -17,15 +25,66 @@ private String extractCommandWord(String input) {
     }
 
 
+    public Command getCommand(String line) {
+        String commandWord = extractCommandWord(line);
+
+        switch (commandWord.toLowerCase()) {
+        case AddStudentCommand.COMMAND_WORD:
+            return new AddStudentCommand(); // Calls delete command handling method
+
+        case DeleteStudentCommand.COMMAND_WORD:
+            return new DeleteStudentCommand(); // Calls add command handling method
+
+        case FindStudentCommand.COMMAND_WORD:
+            return new FindStudentCommand(); // Lists all students
+
+        case ListStudentCommand.COMMAND_WORD:
+            return new ListStudentCommand(); // Lists all students
+
+        case ExitCommand.COMMAND_WORD:
+            return new ExitCommand(); // Lists all students
+
+        default:
+            LOGGER.warning(UNKNOWN_COMMAND_ERROR_MESSAGE);
+            return null; // or handle unknown command appropriately
```

Comment:
can just throw invalid command and it'll auto get handle by current main loop

Reply:
Thank you, I have implemented your suggestion.